db/test/test_dataset.py

```diff
@@ -169,6 +169,61 @@ def test_delete_shapshot(self):
         with self.assertRaises(db.exceptions.NoDataFoundException):
             dataset.get_snapshot(snap_id)
 
+    def test_add_recordings(self):
+        id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        test_data_recordings = {
+            "class_name": "Class #1",
+            "recordings": ["1c085555-3805-428a-982f-e14e0a2b18e6"]
+        }
+        originaldataset = dataset.get(id)
+        dataset.add_recordings(test_data_recordings, id)
+        updateddataset = dataset.get(id)
+        self.assertDictEqual(originaldataset["classes"][1], updateddataset["classes"][1])
+        self.assertEqual(len(originaldataset["classes"][0]["recordings"])+1, len(updateddataset["classes"][0]["recordings"]))
```

Comment:
I said in the previous review that we should check the contents of the dataset, not the length. If you check the contents, you don't need to also check the length. 

For this test, it would be ok to be explicit about the recordings that you expect:

```python
expected_recordings = ["1c085555-3805-428a-982f-e14e0a2b18e6", "0dad432b-16cc-4bf0-8961-fd31d124b01b", "19e698e7-71df-48a9-930e-d4b1a2026c82"]
self.assertEqual(expected_recordings, updateddataset[0]["recordings"])
```

Because you know exactly what should be in the dataset, it's OK to list the recording ids in the test code.